### ISSUE TITLE
README.rst provides bad example

### ISSUE THREAD

**Issue by sweetness-magoo** [cloudtools/awacs#72]
When this code (that is found in the README) is run...

```from awacs.aws import Action, Allow, Policy, Principal, Statement
from awacs.iam import ARN as IAM_ARN
from awacs.s3  import ARN as S3_ARN

account = "123456789012"
user = "user/Bob"

pd = Policy(
    Version="2012-10-17",
    Id="S3-Account-Permissions",
    Statement=[
        Statement(
            Sid="1",
            Effect=Allow,
            Principal=Principal("AWS", [IAM_ARN(account, user)]),
            Action=[Action("s3", "*")],
            Resource=[S3_ARN("my_corporate_bucket/*"),],
        ),
    ],
)
print(pd.to_json())
```
It does not produce the json policy as specified in the README.  The IAM_ARN is wrong. 

Expected: `"arn:aws:iam:123456789012:user/Bob:"`
Actual: `"arn:aws:iam:user/Bob::123456789012"`

**Comment by mprince** [comment]
@sweetness-magoo this should be address by https://github.com/cloudtools/awacs/pull/71

**Comment by markpeek** [comment]
Thanks for the issue. Closed via #71
